python/tskit/trees.py

```diff
@@ -1194,10 +1194,155 @@ def draw_text(self, orientation=None, **kwargs):
+
+        .. code-block:: css
+
+            .tree .symbols .nodes .sample {fill: blue}
+            .tree .labels .nodes text.sample {transform: rotate(90deg)}
+            .tree .labels .nodes text:not(.sample) {visibility: hidden}
+
+        Specific nodes can be targetted by number. The following style will display
+        node 10 in red, and also colour in red the edges whose parent is node 10:
+
+        .. code-block:: css
+
+            .tree .symbols .nodes .n10 {fill: red}
+            .tree .edges .p10 {stroke: red}
+
+        Mutations can be targetted by id, site id, or node number. The following
+        style displays all mutations immediately above node 10 as yellow with a black
+        border
+
+        .. code-block:: css
+
+            .tree .symbols .mutations .n10 {fill: yellow; stroke: black}
+
+        :param str path: The path to the file to write the output. If None, do not
+            write to file.
+        :param size: A tuple of (width, height) giving the width and height of the
+            produced SVG drawing in abstract user units (usually interpreted as pixels on
+            initial display).
+        :type size: tuple(int, int)
+        :param str tree_height_scale: Control how height values for nodes are computed.
+            If this is equal to ``"time"`` (the default), node heights are proportional
+            to their time values. If this is equal to ``"log_time"``, node heights are
+            proportional to their log(time) values. If it is equal to ``"rank"``, node
+            heights are spaced equally according to their ranked times.
+        :param str,float max_tree_height: The maximum tree height value in the current
+            scaling system (see ``tree_height_scale``). Can be either a string or a
+            numeric value. If equal to ``"tree"`` (the default), the maximum tree height
+            is set to be that of the oldest root in the tree. If equal to ``"ts"`` the
+            maximum height is set to be the height of the oldest root in the tree
+            sequence; this is useful when drawing trees from the same tree sequence as it
+            ensures that node heights are consistent. If a numeric value, this is used as
+            the maximum tree height by which to scale other nodes.
+        :param node_labels: If specified, show custom labels for the nodes
+            (specified by ID) that are present in this map; any nodes not present will
+            not have a label.
+        :type node_labels: dict(int, str)
+        :param mutation_labels: If specified, show custom labels for the
+            mutations (specified by ID) that are present in the map; any mutations
+            not present will not have a label.
+        :type mutation_labels: dict(int, str)
+        :param dict root_svg_attributes: Additional attributes, such as an id, that will
+            be embedded in the root ``<svg>`` tag of the generated drawing.
+        :param str style: A
```

Comment:
should say here that this stuff applies to the entire doc it's in

Reply:
I've put this as a Note in #555: 

```
        ..note::
             A feature of SVG style commands is that they apply not just to the contents
             within the <svg> container, but to the entire file. Thus if an SVG file is
             embedded in a larger document, such as an HTML file (e.g. when an SVG
             is displayed inline in a Jupyter notebook), the style will apply to all SVG
             drawings in the notebook. To avoid this, you can tag the SVG with a unique
             SVG using ``root_svg_attributes={'id':'MY_UID'}``, and prepend this to the
             style string, as in ``#MY_UID .tree .edges {stroke: gray}``.
```